benchmarks/ssr/README.md

```diff
@@ -0,0 +1,14 @@
+# Vue.js SSR benchmark
+
+This benchmark renders a table of 10000 rows with 100 columns (1 million cells). It is extremely likely that you will never render 1 million table cells, this benchmark is to demonstrate the overall speeds of Vue.js SSR and time to content comparison between `renderToString` and `renderToStream`.
+
+To view the results follow the run section. Note that the overall completion time for the results are variable, this is due to other system related variants at run time (available memory, processing ect). In ideal circumstances both should finish within equal times.
+
+`renderToStream` pipes the content through a stream which gives massive performance benefits over renderToString. This can be observed through the benchmark.
+
+### run
+
+``` bash
+VUE_ENV=server node renderToString.js
```

Comment:
Also add `NODE_ENV=production` here - probably make it an npm script to run both.